docusaurus/docs/core-concepts/clients/tunnelers/04-macos.md

```diff
@@ -1,3 +1,41 @@
 # MacOS
 
-The MacOS tunneler is currently in preview. Find the app in the [Mac App Store](https://apps.apple.com/us/app/ziti-tunnel/id1460484572?mt=12)
+## Installation
+
+Install the **Ziti Desktop Edge** from the [Mac App Store](https://apps.apple.com/us/app/ziti-tunnel/id1460484572?mt=12)

```

Comment:
Ziti Desktop Edge for Mac

Reply:
nevermind - that's what it's called in the mac store - so let's keep it here as just "Ziti Desktop Edge".